### PR TITLE
Unit Tests: Update on Missing or Incomplete Unit Tests

### PR DESCRIPTION
**Pull request by Bjoern-Steinhagen**
**Current status:** 155 passed, 9 skipped

**9 skipped:**

- test_DesignOverview.py (s) _(Bugfix G-30112)_
- test_ModelCheck.py (s) _(Method does not exist: model_check__get_object_groups_operation)_
- test_SectionDialogue.py (s) _(Method does not exist: create_section_favorite_list)_
- test_concreteDesign.py (s) _(Type does not exsit: concrete_durability)_
- test_objectInformation.py (sss) _(Type does not exist: array_of_get_center_of_gravity_and_objects_info_elements_type)_
- test_steelEffectiveLengths.py (s) _(Bugfix G-30112)_
- test_nodalLoad.py (Assert commented out) _(Bugfix G-30467)_

### CODE REVIEW

**Review by OndraMichal — APPROVED** [review]
Issue with SpectralAnalysisSettings parameter signed_results_using_dominant_mode waits on Olga.

**Comment by OndraMichal** [comment]
All tests missing asserts are covered. Skipped were only those that pytest ignore (@pytest.mark.skipif).

File: UnitTests/test_LineLoads_Test.py [hunk 1]
```diff
@@ -199,4 +308,4 @@ def test_line_set_loads():
     assert Model.clientModel.service.get_line_set_load(1, 1).magnitude == 1200.5
     assert Model.clientModel.service.get_line_set_load(2, 1).magnitude == 2500
     assert Model.clientModel.service.get_line_set_load(3, 1).mass_global == 3100
-    assert Model.clientModel.service.get_line_set_load(4, 1).magnitude == 4000
+    assert Model.clientModel.service.get_line_set_load(4, 1).magnitude == 4000
```
[the remark this answers is not in the document]
Comment (Bjoern-Steinhagen) [reply UnitTests/test_LineLoads_Test.py]: Empty line added at end

File: UnitTests/test_SpectralSettings_test.py [hunk 2]
```diff
@@ -31,3 +31,18 @@ def test_spectral_analysis_settings():
     SpectralAnalysisSettings(6, 'SpectralSettings_6', PeriodicResponseCombinationRule.CQC, DirectionalComponentCombinationRule.ABSOLUTE_SUM)
 
     Model.clientModel.service.finish_modification()
+
+    assert Model.clientModel.service.get_spectral_analysis_settings(2).combination_rule_for_periodic_responses == 'SRSS'
+    assert Model.clientModel.service.get_spectral_analysis_settings(2).combination_rule_for_directional_components == 'SCALED_SUM'
+    assert Model.clientModel.service.get_spectral_analysis_settings(2).use_equivalent_linear_combination == True
+    assert Model.clientModel.service.get_spectral_analysis_settings(2).signed_results_using_dominant_mode == True
```
[the remark this answers is not in the document]
Comment (Bjoern-Steinhagen) [reply UnitTests/test_SpectralSettings_test.py:40]: Function currently deactivated. Assert commented out for now

File: UnitTests/test_SpectralSettings_test.py [hunk 3]
```diff
@@ -31,3 +31,18 @@ def test_spectral_analysis_settings():
     SpectralAnalysisSettings(6, 'SpectralSettings_6', PeriodicResponseCombinationRule.CQC, DirectionalComponentCombinationRule.ABSOLUTE_SUM)
```
[the remark this answers is not in the document]
Comment (Bjoern-Steinhagen) [reply UnitTests/test_SpectralSettings_test.py:33]: Added in the sprectralAnalysisSettings.py

File: UnitTests/test_modalAnalysis_test.py [hunk 4]
```diff
@@ -60,7 +60,15 @@ def test_modal_analysis_settings():
     # Load Case Modal
     LoadCase(2, 'MODAL',params=modalParams)
 
-    #Calculate_all() # Don't use in unit tests. See template for more info.
-
     Model.clientModel.service.finish_modification()
 
+    assert Model.clientModel.service.get_modal_analysis_settings(1).acting_masses_about_axis_x_enabled == False
```
[the remark this answers is not in the document]
Comment (Bjoern-Steinhagen) [reply UnitTests/test_modalAnalysis_test.py]: Asserts are now passing :-)

**Comment by Bjoern-Steinhagen** [comment]
See main comment at top of pull request regarding status of skipped unit tests.

**Comment by OndraMichal** [comment]
158 passed, 7 skipped in 199.21s version 6.02.0023.119.f2143bd6214

**Pull request merged**